### PR TITLE
mfoc & mfcuk & libfreefare: fix <download url&sha256>

### PR DESCRIPTION
**Pull request by m1a0yu3**
due to the Google Code Project will be turned down in early 2016.
https://code.google.com/archive/p/nfc-tools/ has moved to https://github.com/nfc-tools/
this pull request modify the download url and sha256  of the nfc tools

### CODE REVIEW

File: Library/Formula/libfreefare.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Libfreefare < Formula
   desc "API for MIFARE card manipulations"
-  homepage "https://code.google.com/p/libfreefare/"
-  url "https://libfreefare.googlecode.com/files/libfreefare-0.4.0.tar.bz2"
-  sha256 "bfa31d14a99a1247f5ed49195d6373de512e3eb75bf1627658b40cf7f876bc64"
+  homepage "https://github.com/nfc-tools/libfreefare/"
+  url "https://github.com/nfc-tools/libfreefare/releases/download/libfreefare-0.4.0/libfreefare-0.4.0.tar.bz2"
+  sha256 "BFA31D14A99A1247F5ED49195D6373DE512E3EB75BF1627658B40CF7F876BC64"
```
Comment (DomT4) [inline Library/Formula/libfreefare.rb]: Checksums should not be uppercase :smiley:

Comment (m1a0yu3) [inline Library/Formula/libfreefare.rb]: sorry for that  : P
I will modify them

**Comment by DomT4** [comment]
Apparently when they moved they let GitHub regenerate the tag tarballs automatically rather than moving existing ones across. The new tarballs will require the autotools and that build step for `mfcuk` and `mfoc`.

**Comment by apjanke** [comment]
Closing this due to inactivity/lack of response. If you'd still like to pursue this, feel free to respond to the previous comments/questions and re-open on the new [`homebrew/core` repo](https://github.com/homebrew/homebrew-core).

**Pull request closed without merging**